### PR TITLE
added refs, pvt vuln field

### PR DESCRIPTION
**Pull request by yzainee**

### CODE REVIEW

File: src/cve.py [hunk 1]
```diff
@@ -44,16 +45,16 @@ def create_pv_nodes(self):
...
             query = GraphPopulator.construct_graph_nodes(epv_dict)
             success, json_response = BayesianGraph.execute(query)
             e = epv_dict.get('ecosystem')
             p = epv_dict.get('name')
             v = epv_dict.get('version')
-            # Fetch the value of the latest_version from the query created
-            latest_version = "-1"
-            if "latest_version" in query:
+            # Fetch the value of the latest_version from the query create
+            if latest_version == "-1" and "latest_version" in query:
```
Comment (deepak1725) [inline src/cve.py]: latest_version == "-1" isn't this always True..?

Comment (yzainee) [reply src/cve.py]: Its done so that the fetching of value happens once and not for all affected versions of a package

Comment (yzainee) [reply src/cve.py]: just to make sure that these lines execute only once
data = query.split("\'latest_version\'")[1].split(");")[0]
                latest_version = data.replace(",", "").strip().replace("'", "")

Comment (deepak1725) [reply src/cve.py]: That's perfectly fine..  but I can't see any instance when `latest_version == "-1" ` triggers to False

Comment (deepak1725) [reply src/cve.py]: okay

**Pull request merged**